GettingStartedGuides/HoloLens2_GettingStarted/README.md

```diff
@@ -9,6 +9,9 @@ urlFragment: HoloLens2_GettingStarted
 ---
 # Get started with WebView2 in HoloLens 2 Unity apps
 
+> [!IMPORTANT]
+> WebView2 continues to be available. However, support for WebView2 is discontinued (no bug fixes, content updates or technical support), and we can therefore make no guarantees that applications will continue to work for any extended period of time.

```

Comment:
```suggestion
> WebView2 continues to be available. However, support for WebView2 on Hololens is discontinued (no bug fixes, content updates or technical support), and we can therefore make no guarantees that applications will continue to work for any extended period of time.

```

Reply:
@lolambean requesting small change to make it more clear that this is only referring to support on Hololens.

Reply:
Perfect, that is this great suggestion.  I will make this change every where.

Reply:
@champnic I thought I accepted that change, but it doesn't look like it.  I am not sure what I am supposed to do next.  Please let me know.  This is a workflow I don't have a lot of experience with.